applications/DEMApplication/custom_processes/apply_kinematic_constraints_process.hpp

```diff
@@ -1,283 +0,0 @@
-//
-//   Author:   Miguel Angel Celigueta
-//
-
-#if !defined(KRATOS_APPLY_KINEMATIC_CONSTRAINTS_PROCESS )
-#define  KRATOS_APPLY_KINEMATIC_CONSTRAINTS_PROCESS

```

Comment:
It seems like this file was renamed to .h.  I think hpp is the actual extension for C++, while .h is inherited from plain C.
I know that we usually use .h files, but I was trying to slowly change the trend. With this name change, the history of the file is lost.

Reply:
You are right!